portafly/portafly_api.yml

```diff
@@ -68,35 +88,147 @@ components:
           type: integer
       example:
         id: 1
-        created_at: 2019-10-18T05:13:26Z
-        updated_at: 2019-10-18T05:13:27Z
+        created_at: "2019-10-18T05:13:26.000Z"
```

Comment:
Let's try not to include modifications to the previous spec, if there are not necessary. IF they are, include them in a separated commit.